src/main/kotlin/view/TopBar.kt

```diff
@@ -0,0 +1,67 @@
+package view
+
+import controller.ImageController
+import javafx.scene.control.Alert
+import javafx.stage.FileChooser
+import models.IPEwGImage
+import models.ImageModel
+import tornadofx.*
+
+
+class TopBar : View() {
+    val controller: ImageController by inject()
+
+    override val root = menubar {
+        menu("File") {
+            item("Import...") {
+                action {
+                    selectImage()
+                }
+            }
+            item("Export...")
+            item("Quit")
+        }
+        menu("View") {
+            item("Basic Action")
+            item("Advanced")
+            item("Advanced")
+        }
+        menu("Help") {
+            item("How to")
+        }
+    }
+
+    fun selectImage() {
+        val filter = arrayOf(
+            FileChooser.ExtensionFilter(
+                "PNG files (*.png)",
+                "*.png"
+            ),
+            FileChooser.ExtensionFilter(
+                "Bitmap files (*.bmp)",
+                "*.bmp"
+            ),
+            FileChooser.ExtensionFilter(
+                "JPEG files (*.jpeg, *.jpg)",
+                "*.jpeg",
+                "*.jpg"
+            )
+        )
+        val dir = chooseFile(
+            "Select image",
+            filters = filter,
+            mode = FileChooserMode.Single
+        )
+
+        try {
+            controller.load(if (dir.toString() == "[]") "" else "file:///" + dir[0].toString())
```

Comment:
I think we should run this load function only if `dir` is not empty, which means if the `FileChooser` closed without selecting any image, this code should not run at all so no error popup should occur.